src/AccordionItemTitle/accordion-item-title.js

```diff
@@ -48,6 +48,22 @@ class AccordionItemTitle extends Component {
             },
         );
 
+        if (role === 'tab') {
+            return (
+                <div // eslint-disable-line jsx-a11y/no-static-element-interactions
+                    id={id}
+                    aria-selected={expanded}
+                    aria-controls={ariaControls}
+                    className={titleClassName}
+                    onClick={onClick}
+                    role={role}
+                    tabIndex="0"
+                    onKeyPress={this.handleKeyPress}
+                >
+                    {children}
+                </div>
+            );
+        }
```

Comment:
I can't see a difference between this and the code below, returned when `role` is not `tab`.

Reply:
@thibaudcolas The difference is `aria-selected` because `aria-expanded` and I couldn't find a way to inject this dynamically without this big if/else.